evalme/metrics.py

```diff
@@ -141,13 +142,15 @@ def symmetrize(a, b):
         # get metric params
         params = project.get("metric_params", {})
         # remove backup parameters
```

Comment:
I think we should move (or remove) this commentary

Reply:
Fixed